src/AffineVault.sol

```diff
@@ -0,0 +1,508 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity =0.8.16;
+
+import {AccessControlUpgradeable} from "@openzeppelin/contracts-upgradeable/access/AccessControlUpgradeable.sol";
+import {Math} from "@openzeppelin/contracts/utils/math/Math.sol";
+
+import {ERC20} from "solmate/src/tokens/ERC20.sol";
+import {SafeTransferLib} from "solmate/src/utils/SafeTransferLib.sol";
+
+import {AffineGovernable} from "./AffineGovernable.sol";
+import {BridgeEscrow} from "./BridgeEscrow.sol";
+import {BaseStrategy as Strategy} from "./BaseStrategy.sol";
+import {uncheckedInc} from "./libs/Unchecked.sol";
+
+contract AffineVault is AffineGovernable, AccessControlUpgradeable {
```

Comment:
This has a like, 90% code duplication with BaseVault. Why can't we inherit from Basevault and add additional mint, diposit functions instead?

We can se bridgeEscrow and wormholeRouter to 0x0 in the baseInitializer of BaseVault.